### DISCUSSION TITLE
agent config SW_JDBC_TRACE_SQL_PARAMETERS

### DISCUSSION THREAD

**Discussion by wangan0915** [apache/skywalking#10619] [Q&A] [answered]
i set SW_JDBC_TRACE_SQL_PARAMETERS true, my projectA with mybatis could show sql param in trace like this,
![image](https://user-images.githubusercontent.com/39958329/228774265-32c3d607-13df-46df-af8e-9afd3e13a80e.png)

but my projectB with jpa could not show sql param in trace,I wonder if jpa is not supported，Looking forward to your answer，thanks。

**Comment by wu-sheng** [comment, marked as answer]
We don't care about JPA or any framework of ORM. The parameters are collected through supported JDBC driver.
